butter-cms.json

```diff
@@ -34,15 +34,15 @@
     "Image Facedetection": "",
     "Optimized Image Encoding": "",
     "Asset fingerprinting": "",
-    "CDN Support": "No",
+    "CDN Support": "Yes",
     "Antivirus Scanning": "No",
     "Custom Binary Handler": "No",
     "System Requirements": "NA",
     "Supported Databases": "NA",
     "Clustering": "NA",
     "Docker Support": "NA",
     "Backup Feature": "No",
-    "Import/Export": "No",
+    "Import/Export": "Yes",
```

Comment:
![Screen Shot 2019-06-27 at 9 39 54 AM](https://user-images.githubusercontent.com/48689025/60280163-a0797280-98bf-11e9-9500-5c2195200872.png)

Reply:
You can also do this programmatically through their READ/WRITE API

Reply:
Thanks for the link. The entry is mainly about a dedicated import/export feature to get all contents out/in of the system.
The mentioned feature is imho mainly for migration from other vendors to buttercms.